aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -19,6 +19,100 @@
 namespace at {
 namespace native {
 
+template<class scalar_t>
+void cusolver_LU(int m, int n, scalar_t* dA, int ldda, int* ipiv, int* info);
```

Comment:
Really sorry to bugging about this, but the naming scheme is inconsistent. Can you rename this to `cusolverLu`?